spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsCounterService.java

```diff
@@ -54,8 +60,11 @@ public MetricsCounterService(MeterRegistry meterRegistry, MetricsTransmitter met
 
     public void increment(TemporaryMetricsAggregation tma) {
         logBeginningOfIncrementation(tma);
-        int currentCount = incrementCounterAndReturnCurrentValue(tma);
-        Metric metricToTransmit = createMetricToTransmit(tma);
+
+        LocalDate now = LocalDate.now();
```

Comment:
Maybe it would be more clear to call the variable `today`, as it does not contain any time information.